Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.9
+// swift-tools-version:6.0
```

Comment:
Originally the version number needed to be immediately after the colon, but over time the whitespace requirements have been relaxed, allowing for them to be separated with a space.

```suggestion
// swift-tools-version: 6.0
```